src/Pages/jieunyang/Main/Main.scss

```
@@ -0,0 +1,292 @@
+body {
+  width: 100%;
+}
+
+.header {
+  position: fixed;
+  width: 100%;
+  z-index: 5;
+  background-color: white;
+  border: 1px solid lightgray;
+}
+
+.header_container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+}
+
+.logo {
+  margin-left: 150px;
+  font-family: "Lobster", cursive;
+  font-size: 25px;
+}
+
+.header_bar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.search_bar {
+  width: 170px;
+  height: 20px;
+  border: 0.5px solid lightgray;
+  border-radius: 3px;
+}
+
+.fa-search {
+  order: -1;
+}
+
+.icons {
+  display: flex;
+  margin-right: 170px;
+}
+
+.fa-home,
+.fa-comments,
+.fa-compass,
+.fa-heart {
+  font-size: 23px;
+  margin-right: 17px;
+}
+
+.profile_img {
+  width: 22px;
+  height: 22px;
+  border-radius: 70%;
+  margin-right: 17px;
+}
+
+.main_container {
+  position: relative;
+  width: 900px;
+  height: 1600px;
+}
+
+.feeds {
+  position: absolute;
+  width: 600px;
+  left: 17%;
+  margin-top: 100px;
+  border: 0.5px solid lightgray;
+}
+
+.feeds_header {
+  margin-top: 15px;
+  margin-left: 10px;
+  margin-bottom: 15px;
```

```suggestion
margin: 15px 0 15px 10px;
```
로 줄여볼 수 있습니다-!

전체적으로 확인해보시면서, 위와 같이 2개 이상의 `margin` 속성을 주고 있는 경우, 한 줄로 줄여주세요!